src/components/cards/LearnCard.tsx

```diff
@@ -11,14 +11,13 @@ import { useTheme } from '@/theme';
 import { useRoute } from '@react-navigation/native';
 import { analyticsV2 } from '@/analytics';
 
-export const LEARN_CARD_HEIGHT = 184;
-
 type LearnCardProps = {
   cardDetails: LearnCardDetails;
+  rotate?: () => void;
```

Comment:
Nit: rotate is a specific use case for the press handler here, better name is probably `onPress` for consistency.